app/models/user.rb

```diff
@@ -268,7 +268,7 @@ def processed_website_url
   end
 
   def remember_me
-    true
+    Rails.env.test? ? false : true
```

Comment:
Maybe you can mock this method in the spec... to avoid having "test logic" in the actual app

Reply:
Yeah, let me do that.

Reply:
Actually I thought you were referring to something else but I went ahead and removed that. 😆